### ISSUE TITLE
[Bug]: Picker not usable with Voiceover on iOS

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#1766]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-picker

### Expected behavior

Should be able to be used with Voiceover. 
* When the picker is opened focus should go into the picker
* swipe left/right should move to the previous/next picker entries
* double tap should then select the entry

### Actual behavior

When you swipe right VO focus gets bounced back to the first entry in the picker

### What browsers are you seeing the problem in?

Safari

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/picker
2. Start VO
2. Focus on a picker
3. Double Tap to open
4. Swipe right to move to the next item
5. See focus bounce back to first item

**Comment by jnurthen** [comment]
Note - reproduces on iOS 14.5 and 15 - also reproduces on https://github.com/adobe/spectrum-web-components/pull/1759
